### PR TITLE
Changes required for VirtualController EntitySystems

### PR DESCRIPTION
**Pull request by gradientvera**
## About the PR 
Content fixes. Requires https://github.com/space-wizards/RobustToolbox/pull/2424

cc @metalgearsloth

### CODE REVIEW

File: Content.Server/Physics/Controllers/ConveyorController.cs [hunk 1]
```diff
@@ -13,14 +13,13 @@ namespace Content.Server.Physics.Controllers
...
-
-        public override List<Type> UpdatesAfter => new() {typeof(MoverController)};
+        [Dependency] private readonly ConveyorSystem _conveyor = default!;
 
         public override void Initialize()
         {
             base.Initialize();
-            _conveyor = EntitySystem.Get<ConveyorSystem>();
+
+            UpdatesAfter.Add(typeof(MoverController));
```
Comment (PaulRitter) [inline Content.Server/Physics/Controllers/ConveyorController.cs:22]: adding the updatesafter after the basecall which uses it. :sus:

move it infront of the basecall

File: Content.Server/Physics/Controllers/PullController.cs [hunk 2]
```diff
@@ -33,15 +34,13 @@ public class PullController : VirtualController
...
-
-        public override List<Type> UpdatesAfter => new() {typeof(MoverController)};
+        [Dependency] private readonly SharedPullingSystem _pullableSystem = default!;
 
         public override void Initialize()
         {
             base.Initialize();
 
-            _pullableSystem = EntitySystem.Get<SharedPullingSystem>();
+            UpdatesAfter.Add(typeof(MoverController));
```
Comment (PaulRitter) [inline Content.Server/Physics/Controllers/PullController.cs:43]: same here

**Review by PaulRitter — CHANGES_REQUESTED** [review]

File: Content.Server/Physics/Controllers/ConveyorController.cs (hunk 1 above)
Comment (Zumorica) [reply Content.Server/Physics/Controllers/ConveyorController.cs:22]: true, good catch!

Comment (Zumorica) [reply Content.Server/Physics/Controllers/ConveyorController.cs:20]: fixed now

**Review by PaulRitter — APPROVED** [review]

**Pull request merged**